gdbinit.py

```diff
@@ -18,6 +19,42 @@
 
 # Get virtualenv's site-packages path
 venv_path = os.environ.get("PWNDBG_VENV_PATH")
+
+
+def calculate_hash(file_path):
+    result = subprocess.run(["sha256sum", file_path], capture_output=True, text=True)
```

Comment:
We should probably use hashlib instead of computing it via external tool that may not be present
```
In [1]: import hashlib

In [2]: hashlib.sha256?
Signature: hashlib.sha256(string=b'', *, usedforsecurity=True)
Docstring: Returns a sha256 hash object; optionally initialized with a string
Type:      builtin_function_or_method

In [3]: x = hashlib.sha256()

In [4]: x.update(b'asdf')

In [5]: x.hexdigest()
Out[5]: 'f0e4c2f76c58916ec258f246851bea091d14d4247a2fc3e18694461b1816e13b'
```